### ISSUE TITLE
As a developer I'd like to provide default_database_source_path option in config.yml file to skip --src flag for import-db command

### ISSUE THREAD

**Issue by marcinmaruszewski** [drud/ddev#2106]
**Is your feature request related to a problem? Please describe.**
In my Drupal 8 development cycle, I don't use migration tools for creating nodes/users/content entities stored in the database on the development environment. Instead of I have a small database dump, so I can have the same content for each developer in the project. 

When I start a new task, switch between branches, messed up something, etc. I use `ddev import-db --src database/database.sql` command to restore it (I also use the `post-import-db` hook to fire some composer/drupal related stuff). Unfortunately, I need to pass the path to database dump with `--src` flag all the time.

**Describe the solution you'd like**
I'd like to provide some `default_database_source_path` option in `config.yml` so I can just type `ddev import-db`.

**Describe alternatives you've considered**
I could create custom `restore-database` command/script in `.ddev/commands/host`, but I think it easier to maintain it in the project configuration file.

**Comment by rfay** [comment]
Thanks. You know about the many other ways to import, right? `ddev mysql` and `ddev restore-snapshot` ? People love snapshotting for switching branches. If you always snapshot with the same name (maybe after deleting existing) you can restore with same name...

**Comment by marcinmaruszewski** [comment]
Hey @rfay, thanks for such a quick response.

The `ddev restore-snapshot` command seems quite good, but as far as I can see it saves the `db` container volume and put it inside `.ddev/db_snapshots` directory. I see that this directory can take about 300 MB while `database.sql.tar.gz` file is only 15 MB. Since we store this file inside our development git repository it's not possible to use `ddev snapshot` and `ddev restore-snapshot` commands.

About the `ddev mysql` command. It's not suitable for databases dumped in *.sql.gz archives. I'd also have to remember eather to put some database drop/create command at the beginning of this SQL file, or add it to `pre-mysql` hook (but it would be risky). I'd also still have to pass the database file location.

To sum up, all that I've written - those commands are great, but not feasible in my situation. Please reconsider my request with other developers and the ddev community. If you decided to not include this `default_database_source_path` setting (which is completely understandable) I'll stick to the custom command/script inside `.ddev/commands/host`.

Once again thanks for your comment! Keep up doing great work with contributing to ddev tool :+1:

**Comment by rfay** [comment]
Just saying... `gzip -dc <filename> | ddev mysql db` - ddev mysql can handle gzipped files that way just fine. I understand that doesn't necessarily solve the problem the way you want to solve it though.

**Comment by marcinmaruszewski** [comment]
You are right, I can do it this way! But it's easier to create a custom host command/script.

I'll stick right now to this custom host command/script solution, so please do whatever you like with this issue - postpone it, add some label or just close :white_flag:

**Comment by rfay** [comment]
I think there are enough ways to solve this request that it doesn't need increased configuration options. Thanks for posting the issue, and glad you have a way to do it!
